pkg/kudoctl/cmd/plan/plan_status.go

```diff
@@ -60,12 +60,17 @@ func status(kc *kudo.Client, options *Options, ns string) error {
 		return ""
 	}
 
-	rootDisplay := fmt.Sprintf("%s (Operator-Version: \"%s\" Active-Plan: \"%s\")", instance.Name, instance.Spec.OperatorVersion.Name, lastPlanStatus.Name)
+	rootDisplay := fmt.Sprintf("%s (Operator-Version: \"%s\" Active-Plan: \"%s\", last updated: \"%s\")", instance.Name, instance.Spec.OperatorVersion.Name, lastPlanStatus.Name, instance.Status.AggregatedStatus.LastUpdated.Format("2006-01-02 15:04:05"))
```

Comment:
Not a review comment, just a "This is how golang does date formatting" comment.... 🤦‍♂